docs/customization/factory.rst

```diff
@@ -0,0 +1,131 @@
+Customizing Factories
+=====================
+
+.. warning::
+
+    In **Sylius** we are already decorating factories from components in Core.
+    Often you will be needing to add your very own methods to them. You need to check before which factory is your resource using.
```

Comment:
Scratch the first sentence - it's irrelevant in the warning scope.
Furthermore, the why, and how are explained right below.
`before which` => `beforehand, which`